diag_manager/README.md

```diff
@@ -322,8 +311,7 @@ diag_files:
     corner3: -60, 0
     corner4: -60, 75
 - file_name: normal2
-  freq: -1
-  freq_units: days
+  freq: -1 days
```

Comment:
does this actually need units since its freq is -1?  Same question with 0.

Reply:
The code was updated to not require units if the freq is -1 or 0
https://github.com/NOAA-GFDL/FMS/pull/1269/commits/bf431c619914240409917eaef51c3e69127bf8da